packages/core/platform/platform.js

```diff
@@ -220,27 +74,19 @@ const parseUserAgent = (userAgent) => {
 		deprecate({name: `supporting ${detectedInfo.browserName} version before ${supportedVersions[detectedInfo.browserName]}`, until: '5.0.0'});
```

Comment:
@0x64 Why the version is already '5.0.0'? `until` is also unclear for me.

Reply:
`until` makes a message like " Will be removed in ${until}" and the original message was correct. But, it looks not good to notify 'deprecate' along with versions as we will update the base browser versions annually.
For now, it looks better simply removing 'until' option.